Volha_Ivanova/Module_1_Java/Java_Fundamentals/HomeWork1/src/com/epam/cdp/volha_ivanova/task3/Calculator.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.cdp.volha_ivanova.task3;
+
+public class Calculator {
+	
+	private int value1;
+	private int value2;
+	
+	  public Calculator(int value1, int value2) {
```

Comment:
Calculator shouldn't has the parameters.
By your implementation you should create new object of calculator each time to calculate something. What about memory or performance?
Your should have one single object of calculator in your runner and your methods should have arguments